bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/api/MyRenaultHttpSession.java

```diff
@@ -224,6 +229,67 @@ public void getLocation(Car car)
         }
     }
 
+    public void actionHvacOn(Double hvacTargetTemperature)
+            throws RenaultForbiddenException, RenaultNotImplementedException {
+        Request request = httpClient
+                .newRequest(this.constants.getKamereonRootUrl() + "/commerce/v1/accounts/" + kamereonaccountId
+                        + "/kamereon/kca/car-adapter/v1/cars/" + config.vin + "/actions/hvac-start?country="
+                        + getCountry(config))
+                .method(HttpMethod.POST).header("Content-type", "application/vnd.api+json")
+                .header("apikey", this.constants.getKamereonApiKey())
+                .header("x-kamereon-authorization", "Bearer " + kamereonToken).header("x-gigya-id_token", jwt);
+        request.content(new StringContentProvider(
+                "{\"data\":{\"type\":\"HvacStart\",\"attributes\":{\"action\":\"start\",\"targetTemperature\":\""
+                        + hvacTargetTemperature + "\"}}}",
+                "utf-8"));
+        try {
+            ContentResponse response = request.send();
+            logger.debug("Kamereon Response HVAC ON: {}", response.getContentAsString());
+            if (HttpStatus.OK_200 != response.getStatus()) {
+                logger.warn("Kamereon Response: [{}] {} {}", response.getStatus(), response.getReason(),
+                        response.getContentAsString());
+                if (HttpStatus.FORBIDDEN_403 == response.getStatus()) {
+                    throw new RenaultForbiddenException(
+                            "Kamereon Response Forbidden! Ensure the car is paired in your MyRenault App.");
+                } else if (HttpStatus.NOT_IMPLEMENTED_501 == response.getStatus()) {
+                    throw new RenaultNotImplementedException(
+                            "Kamereon Service Not Implemented: [" + response.getStatus() + "] " + response.getReason());
+                }
```

Comment:
Nothing if other status ?

Reply:
I always log.warn if not HTTP 200.  I only throw errors for known situations like not implemented or forbidden.

Reply:
OK